### PR TITLE
PR: Complement to proxy style fix on spyder

### LINKED ISSUE #191: Dropdown menus are not looking well in PyQt 5.12
### Describe Your Environment

Please, report the output of the command (from v2.6+) or fill the information below:

- QDarkStyle: 2.7
- OS: Linux 
- Python: 3.6
- PyQt: 5.12
- QtPy: 1.8
- QT_API: pyqt5

### Language

Python

### Description / Steps to Reproduce [if necessary]

All entries in dropdown menus have a big blank space to its left when using PyQt 5.12

### Actual Result

Selección_012

### Expected Results / Proposed Result

This is how things look in PyQt 5.9, which is fine

Selección_013

### LINKED ISSUE #193: What to include in package - data, example, scripts, tests
Considering the issue raised by @goanpeca  in #188, some things need to be updated. 

There is a MANIFEST.in in the repo:

- We can update it, following some better rules such as [this](https://blog.ionelmc.ro/presentations/packaging/#slide:15) and add [check for manifest](https://github.com/mgedmin/check-manifest);

- Remove MANIFEST and use [setuptools_scm](https://github.com/pypa/setuptools_scm);

- Use MANIFEST.in + include_package_data=True;

I'll study this a little bit more and bring some more info. :)

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #191
Related to #169

### CODE REVIEW

**Comment by goanpeca** [comment]
@dpizetta not sure about the CI fail

**Comment by goanpeca** [comment]
```

QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-travis'
DEBUG:qdarkstyle:Reading QSS file in: :qdarkstyle/style.qss
INFO:qdarkstyle:QSS file sucessfuly loaded.
DEBUG:qdarkstyle:Checking patches for being applied.
INFO:qdarkstyle:Found application patches to be applied.
INFO:__main__:QDarkStyle Example - (QDarkStyle=v2.8, QtPy=v1.8.0, PySide2=v5.13.0, Qt=v5.13.0, Python=v3.6.7)
Traceback (most recent call last):
  File "example/example.py", line 376, in <module>
    sys.exit(main())
  File "example/example.py", line 178, in main
    ui_buttons.toolButtonDelayedPopup.menu().addAction(action)
AttributeError: 'NoneType' object has no attribute 'addAction'
ERROR: InvocationError for command /home/travis/build/ColinDuquesnoy/QDarkStyleSheet/.tox/py36-pyside2/bin/python example/example.py --qt_from=pyside2 --test (exited with code 1)
```

**Comment by dpizetta** [comment]
> @dpizetta not sure about the CI fail

Oh, sorry my fault. I've updated all UI files now. This should be corrected.

File: setup.py [hunk 1]
```diff
@@ -50,6 +50,7 @@
     entry_points={"console_scripts": ["qdarkstyle=qdarkstyle.__main__:main"]},
     extras_require=extras_require,
     install_requires=install_requires,
+    include_package_data=True,
```
Comment (dpizetta) [inline setup.py]: May we need to change the MANIFEST to include the resources too. In the link I've posted it suggests using general rules such as (not sure about ):

graft qdarkstyle
graft example
graft test
prune .*
prune docs
include *.py *.rst *.ini *.cfg *.scss *.svg *.qrc *.qss
global-exclude *.py[cod] __pycache__ *.so

Not element names, which must be updated if file names are changed.

File: qdarkstyle/qss/_styles.scss [hunk 2]
```diff
@@ -1194,7 +1194,7 @@ QComboBox {
     border-radius: $SIZE_BORDER_RADIUS;
     selection-background-color: $COLOR_SELECTION_NORMAL;
     padding-left: 4px;
-    padding-right: 4px;
+    padding-right: 36px;  /* 4 + 16*2 See scrollbar size */
```
Comment (dpizetta) [inline qdarkstyle/qss/_styles.scss]: Ok, just that? :)

**Comment by dpizetta** [comment]
There is still the error in pyside test run. I'll check, but don't mind about this if others have passed.

File: qdarkstyle/qss/_styles.scss (hunk 2 above)
Comment (goanpeca) [reply qdarkstyle/qss/_styles.scss]: Yes, basically we need to add enough padding to account for the scrollbar size 16*2 + the original padding (4)

So yeah :-)

**Comment by dpizetta** [comment]
@goanpeca do you think it is better to split this PR removing the changes needed in #193 ?  So we can merge everything else.

**Comment by goanpeca** [comment]
@dpizetta just did!

**Pull request merged**